docs/customization/api.rst

```diff
@@ -0,0 +1,80 @@
+Customizing API
+===============
+
+For create all endpoints in Sylius API we are using the API Platform.
+API Platform allows to configure endpoint by ``yaml`` and ``xml`` files or in ``annotations``.
+In these guide we will show you how to customize Sylius API endpoint with ``xml``.
+
+How to prepare project to customization?
+========================================
+
+For any kind of customization firstly you need to configure location of your configs.
+
+.. code-block:: yaml
+    # config/packages/api_platform.yaml
+    api_platform:
+    mapping:
+        paths:
+            - '%kernel.project_dir%/vendor/sylius/sylius/src/Sylius/Bundle/ApiBundle/Resources/config/api_resources'
+            - '%kernel.project_dir%/config/api_platform'
+    patch_formats:
+        json: ['application/merge-patch+json']
+    swagger:
+        versions: [3]
+
+And now if you want to customize any API resource you need copy entire configuration of this resource from ``%kernel.project_dir%/vendor/sylius/sylius/src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/`` to ``%kernel.project_dir%/config/api_platform``
+
+How to add an additional endpoint to a resource?
+================================================
+
+Let's assume that you want to add new endpoint to ``order`` which will dispatch a command.
+You need to add proper configuration in ``config/api_platform/Order.xml``:
+
+.. code-block:: xml
+
+    <collectionOperations>
+        <collectionOperation name="custom_operation">
+            <attribute name="method">POST</attribute>
+            <attribute name="path">/shop/order/custom-operation</attribute>
```

Comment:
```suggestion
            <attribute name="path">/shop/orders/custom-operation</attribute>
```

Well it is example and user can make it `order`, but we are using `orders` so I would stay with it 😄